dipy/align/imaffine.py

```diff
@@ -954,7 +954,7 @@ def _init_optimizer(self, static, moving, transform, params0,
 
     def optimize(self, static, moving, transform, params0,
                  static_grid2world=None, moving_grid2world=None,
-                 starting_affine=None):
+                 starting_affine=None, ret_metric=False):
```

Comment:
@nilgoyette: Thanks for pointing out the thing about 'starting_affine'. Here is the optimize function definition, So the 'starting_affine' is there as an optional parameter but I think it is position independent so passing a value to that argument won't work (please correct me if I am wrong). So, that's why I stick to the name 'starting_affine'. But I have changed it to 'starting_affine=affine' from what it was earlier (aesthetically it looks better now). A very good point though, I really appreciate all the useful comments that you have made, it really helped me to improve the quality of the code. Thanks.